<ui version="4.0" >
 <class>GameHud</class>
 <widget class="QWidget" name="GameHud" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>304</width>
         <height>181</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="ability5" >
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>&lt;hidden></string>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ability4" >
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>&lt;hidden></string>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ability3" >
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>&lt;hidden></string>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ability2" >
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>&lt;hidden></string>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ability1" >
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>&lt;hidden></string>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="playerInfo" >
         <property name="text" >
          <string>&lt;glui></string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="selectedPortrait" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="selectedInfo" >
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>&lt;glui></string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>500</width>
         <height>331</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="chatFrame" >
       <property name="minimumSize" >
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;glui></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="chatInput" >
       <property name="minimumSize" >
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <resources/>
 <connections/>
</ui>
